docs/serving/autoscaling/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Autoscaling"
+linkTitle: "Autoscaling"
+weight: 20
+type: "docs"
+aliases:
+    - /docs/serving/autoscaling/docs/serving/configuring-autoscaling.md
```

Comment:
I think you want this to be:
```suggestion
    - /docs/serving/configuring-autoscaling/
```

So that existing links to https://knative.dev/development/serving/configuring-autoscaling/ will redirect to this page.

Reply:
Thank you, apologies still getting my head around the aliases :slightly_smiling_face: